distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -171,6 +174,50 @@ type Vector a
     new_builder : Integer -> Builder
     new_builder (capacity=10) = Builder.new capacity
 
+    ## PRIVATE
+       ADVANCED
+       Creates a new `Vector` by providing a `Builder` to the provided function.
+
+       A vector `Builder` is a mutable data structure, that allows for gathering
+       a number of elements and then converting them into a vector. This is
+       particularly useful when the number of elements is not known upfront.
+
+       `.build` creates a new `Builder`, passes it to the provided function,
+       which can add elements using the `Builder`'s `.append` method. When the
+       function is done, `.build` then closes the `Builder and returns the
+       resulting `Vector`.
+
+       The provided function should call `.append` to add new elements to the
+       `Builder`. The return value of the provided function is not used, unless
+       it is a dataflow error, in which case the `Vector` is not built, and the
+       dataflow error is propagted instead.
+
+       A vector allows to store an arbitrary number of elements in linear
+       memory. It is the recommended data structure for most applications.
+
+       Arguments:
+       - function: a function taking a `Builder` and using it to add elements.
+       - capacity: Initial capacity of the Vector.Builder
+
+       ! Error Conditions
+
+         - If the provided function throws a dataflow error, the `Vector` is not
+           built, and the error is propagted instead.
+
+       > Example
+         Construct a vector using a builder that contains the items 1 to 5.
+
+             Vector.build builder->
+                 builder.append 1
+                 builder.append 2
+                 builder.append 3
+             # => [1, 2, 3]
+    build : (Builder -> Any) -> Integer -> Vector
+    build (function : Builder -> Any) (capacity : Integer = 10) -> Vector =
```

Comment:
```suggestion
    build (function : Builder -> Any) (initial_capacity : Integer = 10) -> Vector =
```

I think maybe adding `initial_` would clarify the meaning of this variable; without _initial_ it could sound like it's the _max_ capacity.

Reply:
Done